### ISSUE TITLE
test_progress_meter, in test_rms.py, errors when output isn't captured.

### ISSUE THREAD

**Issue by mnmelo** [MDAnalysis/mdanalysis#1412]
Under 0.16.2-dev0, running `mda_nosetests` with the `-s` flag
```
-s, --nocapture       Don't capture stdout (any stdout output will be
                      printed immediately) [NOSE_NOCAPTURE]
```
causes `test_progress_meter` from `test_rms.py` to error:
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/manel/src/MDAnalysis/mdanalysis/testsuite/build/lib.linux-x86_64-2.7/MDAnalysisTests/analysis/test_rms.py", line 170, in test_progress_meter
    actual = sys.stderr.getvalue().strip().split('\r')[-1]
AttributeError: 'file' object has no attribute 'getvalue'
```

This seems related to the `sys.stderr`/`sys.stdout` juggling done in `test_progress_meter` to inspect the output stream, and the fact that `nose` is messing with output too (though it'd seem that *not* capturing output would interfere less).

Perhaps it can be worked around, but I'd also be ok with making this a skippable test:
`@skipif(not hasattr(sys.stderr, 'getvalue')`
Anyway, given the ongoing test migration (#884) I'll leave this open for addressing afterwards.

**Comment by jbarnoud** [comment]
This seems fixed since #1450. @mnmelo could you check if you still experience the issue?

**Comment by mnmelo** [comment]
Indeed, `test_progress_meter` now succeeds, regardless of output capture method (tried with all capture options: `--capture=fd`, `--capture=sys`, and `--capture=no`).

Quite neatly, even if `--capture=no` is set (or its short form `-s`), the test produces no visible output.
